openmdao_book/features/core_features/working_with_components/continuous_variables.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>is the <code>automethod</code>  stuff going to be replaced with something else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I wasn't able to find a Markdown equivalent so we're sticking with autoX::</p>